openmc_plotter/docks.py

```diff
@@ -881,6 +887,10 @@ def updateMaskZeros(self):
         cv = self.model.currentView
         self.maskZeroBox.setChecked(cv.tallyMaskZeroValues)
 
+    def updateVolumeNorm(self):
```

Comment:
One thing came to mine here. If the mesh results are now being volume normalized, we might need to update the tally label based on that setting. The flux units still appear as `Particle-cm/Particle` when this box is checked.

Reply:
Very good point -- I'll get that fixed

Reply:
This has now been implemented :heavy_check_mark: